Main.cs

```diff
@@ -711,5 +749,38 @@ private static string ReadPassword()
                 }
             }
         }
+
+        private static void printHelp()
+        {
+            Console.WriteLine("A wrapper binary that can be used to host executables as an Windows services");
+            Console.WriteLine("");
+            Console.WriteLine("Usage: winsw <command> [<args>], missing arguments trigger the service mode");
+            Console.WriteLine("");
+            printAvailableCommandsInfo();
+            Console.WriteLine("");
+            printVersion();
+            Console.WriteLine("More info: https://github.com/kohsuke/winsw");
+            Console.WriteLine("Bug tracker: https://github.com/kohsuke/winsw/issues");
+        }
+
+        //TODO: Rework to enum in winsw-2.0
+        private static void printAvailableCommandsInfo()
+        {
+            Console.WriteLine("Available commands:");
+            Console.WriteLine("- 'install'   - install the service to Windows Service Controller");
+            Console.WriteLine("- 'uninstall' - uninstall the service");
+            Console.WriteLine("- 'start'     - start the service (must be installed before)");
+            Console.WriteLine("- 'stop'      - stop the service");
+            Console.WriteLine("- 'restart'   - restart the service");
+            Console.WriteLine("- 'status'    - check the current status of the service");
+            Console.WriteLine("- 'test'      - check if the service can be started and then stopped");  
+            Console.WriteLine("- 'version'   - print the version info");
+            Console.WriteLine("- 'help'      - print the help info");
+        }

```

Comment:
What about restart! and redirect.  Redirect could go into optional additional parameters or something like that